MANUAL_REGRESSION.md

```diff
@@ -0,0 +1,99 @@
+# Manual Regression Tests for JS SDK
+
+## Functional
+
+##### 1. Face photo webcam capture
+(on Google Chrome, Firefox, Safari, IE11 and Microsoft Edge browsers)
+
+0. Given webcam is connected to the computer
+1. Go through the flow to face capture
+    - browser should ask to enable the webcam
+2. Accept the webcam to be used on browser
+    - photo capture frame should display preview from webcam
+    - user should be able to upload a picture anyway
+3. Take photo with a webcam
+    - 3-second counter should start and then photo should be taken
+    - confirmation screen should show up containing photo that was taken
+    - user should be able to retake or continue with taken photo
+
+##### 2. Document photo webcam capture
+(on Google Chrome, Firefox, Safari, IE11 and Microsoft Edge browsers)
+
+0. Given webcam is connected to the computer
+1. Open link with additional GET parameter `?useWebcam=true`
+2. Go through the flow to document capture
+    - browser should ask to enable the webcam
+3. Accept the webcam to be used on browser
+    - photo capture frame should display preview from webcam
+    - user should be able to upload a picutre anyway
+4. Place document in front of camera so that it aligns the edges of document capture frame
+    - document should be auto-captured
+    - confirmation screen should show up containing a photo that was taken
+    - user should be able to retake or continue with taken photo
+
+##### 3. Cross-device with SMS
+(on one of the browsers)
+
+0. Given user is upload document page
+1. Click on link to start cross-device flow
+    - user should see `Continue your verification on mobile` screen
+    - user should be able to provide mobile number from any country
+    - user should see the option to send SMS
+    - user should see option to copy link
+2. Type valid mobile number connected to mobile test device and send
+    - user should see `Check your mobile` screen
+    - user should see option to resend link
+    - user should receive SMS on a mobile device
+3. Open link on mobile device
+    - user should see `Upload front of document` screen
+    - user should be able to upload a document from a mobile device
+
+##### 4. Cross-device resend SMS
+(on one of the browsers)
+
+0. Given user is on first page of cross-device flow
+1. Type valid mobile number connected to mobile test device and send
+    - user should see `Check your mobile` screen
+    - user should see option to resend link
+2. Click `Resend link`
+    - user should see `Continue your verification on mobile` screen again
+    - user should be able to provide mobile number again
+    - user should see the option to send SMS
+
+##### 5. Cross-device errors
+(on one of the browsers)
+
+0. Given user is on first page of cross-device flow
+1. Type invalid mobile number and click send
+    - user should see an validation error
+    - user should persist on the same screen
```

Comment:
Sms overuse could be tested by resending the SMS more than 5 times. Not sure if we want to do that on prod link because of SMS cost.
On staging only: SMS errors coming from the server can be tested by providing a US number (+1 234 567 8912)

Reply:
Good tip, done!